openpype/hosts/unreal/api/plugin.py

```diff
@@ -242,6 +221,24 @@ def get_pre_create_attr_defs(self):
         ]
 
 
-class Loader(LoaderPlugin, ABC):
-    """This serves as skeleton for future Ayon specific functionality"""
-    pass
+@six.add_metaclass(ABCMeta)
+class UnrealBaseLoader(LoaderPlugin):
+    """Base class for Unreal loader plugins."""
+    root = "/Game/Ayon"
+    suffix = "_CON"
+
+    def update(self, container, representation):
+        asset_dir = container["namespace"]
+        container_name = container['objectName']
+
+        data = {
```

Comment:
This is optional suggestion, to change the keys to `representation_id` and `version_id`.